.github/workflows/docs.yml

```diff
@@ -0,0 +1,25 @@
+name: Docs
```

Comment:
If the Dir is called website I'd suggest calling this file and name website as well

Reply:
yes, makes sense. Renamed it